lib/features/login/presentation/pages/login_page.dart

```diff
@@ -0,0 +1,192 @@
+import 'package:flutter/material.dart';
+
+import '../../../../utilities/constants.dart';
+import '../../../../utilities/size_config.dart';
+
+class LoginPage extends StatelessWidget {
+  final GlobalKey<FormState> formKey = GlobalKey<FormState>();
+
+  @override
+  Widget build(BuildContext context) {
+    SizeConfig(context); // initializing SizeConfig
+    final double defaultSize = SizeConfig.defaultSize;
+
+    return Scaffold(
+      body: Padding(
+        padding: EdgeInsets.symmetric(
+          horizontal: defaultSize * Constants.kDefaultPaddingMultiplier,
+        ),
+        child: Column(
+          mainAxisAlignment: MainAxisAlignment.center,
+          mainAxisSize: MainAxisSize.max,
+          children: <Widget>[
+            Text(
+              Constants.kAppTitle,
+              style: TextStyle(
+                color: Constants.kTextColor,
+                fontSize: defaultSize * 4,
+              ),
+            ),
+            SizedBox(height: defaultSize),
+            Text(
+              'Log in to find the best trades',
+              style: TextStyle(
+                color: Constants.kTextLightColor,
+                fontSize: defaultSize * 1.6,
+              ),
+            ),
+            SizedBox(
+              height: defaultSize * Constants.kDefaultPaddingMultiplier * 2.5,
+            ),
+            Row(
+              mainAxisAlignment: MainAxisAlignment.spaceBetween,
+              crossAxisAlignment: CrossAxisAlignment.center,
+              children: [
+                FlatButton(
```

Comment:
I think this and all other Flat buttons need ot be wrapped in a ButtonTheme in order to apply a height and minWidth to them

Reply:
Good catch bro. SOrry about the whole master/stable confusion. Fix on [080f66c](https://github.com/rishFilet/swipe_n_swap/pull/2/commits/080f66c0bf4d5553860fccb16068a8fa09ffe2aa)